src/components/Menu/index.styl

```diff
@@ -9,10 +9,10 @@
     position   absolute
     @extend    $popover
     right      0
-    padding    5px 0
+    padding    .3125rem 0
     margin     0
-    margin-top 1px
-    min-width  220px
+    margin-top .0625rem
```

Comment:
I guess we need a padding and margin scale next :p

Reply:
What do you mean?